dev/io.openliberty.reporting.internal/src/io/openliberty/reporting/internal/CVEResponseHandler.java

```diff
@@ -0,0 +1,112 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package io.openliberty.reporting.internal;
+
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import com.ibm.json.java.JSONArray;
+import com.ibm.json.java.JSONObject;
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+
+/**
+ * <p>
+ * CVEResponseHandler checks the response from the cloud service for the
+ * productEdition and outputs a message to the logs.
+ * </p>
+ */
+public class CVEResponseHandler {
+
+	private static final TraceComponent tc = Tr.register(CVEResponseHandler.class);
+
+	private static final String LINE_SEPARATOR = getSystemProperty("line.separator");
```

Comment:
```suggestion
	private static final String LINE_SEPARATOR = System.getProperty("line.separator");
```
`line.separator` is not a protected system property

Reply:
Done.